presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeIdenticalWindows.java

```diff
@@ -190,6 +190,108 @@ public void testIdenticalWindowSpecificationsAAcpA()
     }
 
     @Test
+    public void testIdenticalWindowSpecificationsDefaultFrame()
```

Comment:
for this commit message, replace MIW with text. I think most readers will not know what MIW is referring to.